cluster-sync/sync.sh

```diff
@@ -26,9 +26,10 @@ if [ "${KUBEVIRT_PROVIDER}" != "external" ]; then
 fi
 
 # TODO: fetch it dynamically
-export OPERATOR_VERSION=0.0.1
+export OPERATOR_VERSION=v0.0.1
 
-_kubectl apply -f manifests/vm-import-operator/${OPERATOR_VERSION}/v2v_v1alpha1_resourcemapping_crd.yaml
-_kubectl apply -f manifests/vm-import-operator/${OPERATOR_VERSION}/v2v_v1alpha1_virtualmachineimport_crd.yaml
-_kubectl apply -f manifests/vm-import-operator/${OPERATOR_VERSION}/config_map.yaml
-_kubectl apply -f manifests/vm-import-operator/${OPERATOR_VERSION}/operator.yaml
+_kubectl apply -f https://github.com/kubevirt/vm-import-operator/releases/download/${OPERATOR_VERSION}/namespace.yaml
```

Comment:
We need to deploy the manifests into the same namespace as kubevirt and for that the files need to be processed before being applied.

With local files we can run `TARGET_NAMESPACE=kubevirt make gen-manifests` and we're ready to test.

Reply:
Right, thanks. I've changed it so we use `make gen-manifests`, and always latest operator from code, not latest public version.